server/routes/requests.js

```diff
@@ -74,4 +74,68 @@ router.post('/addRequest', async (req, res, next) => {
     }
 });
 
+router.patch('/updateStatus/:id/:timeOffset', async (req,res, next) => {
+    let alumniId = req.params.id;
+    let timeOffset = parseInt(req.params.timeOffset);
+    let requestId = req.body.requestId;
+    let newStatus = req.body.newStatus;
+    let conditions = ['Awaiting Confirmation', 'Confirmed', 'Completed']
+    let requests = []
+    try {
+        let request = await requestSchema.findById(requestId);
+        let matching_queries = await requestSchema.find({
+            'status': 'Confirmed',
+            'time.id': 'request.id'
+        })
+        if (await (matching_queries === [])) {
+            console.log('Success')
+        }
```

Comment:
I'm a little stuck here - I don't want an alumni to be able to confirm a meeting when they already have one scheduled for the same time - at present I have a warning on the home page, but I'd rather it not let them hit approve. NOTE: Might be easier to accomplish on frontend

Reply:
Filtering now performed on front end